aviary/mission/gasp_based/ode/base_ode.py

```diff
@@ -234,21 +234,20 @@ def AddThrottleControl(
             )
 
     def add_flight_conditions(self, nn, input_speed_type=SpeedType.TAS):
```

Comment:
Now that this adds `Atmosphere` instead of `FlightConditions`, maybe rename this method to `add_atmosphere` or similar?